framework/auth/oauth_scopes.py

```diff
@@ -71,6 +77,9 @@ class ComposedScopes(object):
     USERS_READ = (CoreScopes.USERS_READ,)
     USERS_WRITE = USERS_READ + (CoreScopes.USERS_WRITE,)
 
+    USERS_FULL_READ = USERS_READ + (CoreScopes.USERS_ADDON_READ,)
```

Comment:
Remember: scopes are not about what a user can do. (cookies are an all-access pass)

Scopes are about what you'd let a third-party do on your behalf. (eg importing tokens into a specific project; of course the third party has no way of authorizing google drive without you) 

For now, we may want to revisit putting this in the narrow bucket of `USERS_FULL_READ` vs just the very powerful  `FULL_READ`. User-related scopes were originally envisioned for sensitive profile data, not addon management. (viewing someone's email address shouldn't require being able to authorize addons)